app/locales/fr/translations.js

```diff
@@ -224,6 +224,7 @@ export default {
     'instructors': "Instructeurs",
     'instructorsManageTitle': "Manager Instructeurs",
     'confirmObjectiveRemoval': "Êtes vous sûr de vouloir supprimer cet objectif?",
+    'confirmSessionRemoval': 'Êtes vous sûr de vouloir supprimer cet séance?',
```

Comment:
@saschaben please confirm this translation for `Are you sure you want to delete this session?`